spec/PlayerSpec.js

```diff
@@ -0,0 +1,58 @@
+describe("Player", function() {
```

Comment:
This does not feel like it comes from this project. If you used to have an example, you should delete before sending a PR.